lib/engines/with_migrations.rb

```diff
@@ -0,0 +1,24 @@
+module Engines
+  # It DRYies the common snippet to append engine's migrations to the main app
+  module WithMigrations
```

Comment:
put this in `Shared` engine please, is where all the common stuff goes. Makes sense?